tools/analyzer_plugin/test/integration/diagnostics/callback_ref_test.dart

```diff
@@ -244,6 +353,7 @@ class CallbackRefDiagnosticFnComponentTestNoFix extends CallbackRefDiagnosticTes
 @reflectiveTest
 class CallbackRefDiagnosticClassComponentTest extends CallbackRefDiagnosticWithFixTest {
   static const usageSourceWithinClassComponent = '''
+// @dart=2.11
```

Comment:
Is this for a non-null safe assertion / test somewhere?

Reply:
Oh oops thanks for catching that!